### DISCUSSION TITLE
Ayuda. No funciona el multi url

### DISCUSSION THREAD

**Discussion by Edgardrh** [chamilo/chamilo-lms#5358] [General] [answered]
habilite la funcion de multi url desde el archivo configuration.php y al entrar a chamilo si se habilito la opcion "Configurar acceso a multiple URLs" en el panel de administracion. Ya dentro de esta opcion agrege las 2 urls que necesito pero al activarlas y querer entrar a la url me aparece un error 403.

No se que paso me falta para que pueda funcionar. El chamilo lo instale con Hostgator

**Comment by ywarnier** [comment, marked as answer]
Hola @Edgardrh 
Transferí tu solicitud a una discusión porque no es un reporte de error sino una solicitud de ayuda.
Para que multi-url funcione, tienes que:
- cambiar configuration.php para activar la funcionalidad
- configurar las URLs dentro de Chamilo, pero también
- configurar los vhosts de tu servidor web (no estoy seguro de como hacerlo en hostgator pero diría que con "Editar la zona DNS" si son subdominios) para que las distintas URLs apunten a la misma carpeta donde está tu instalación de Chamilo.

Tienes un poco de información al respecto en inglés aquí: https://docs.chamilo.org/admin-guide/global_features/multi-url/installation

**Reply by Edgardrh** [reply]
Hola, con la ayuda anterior ya funciono mejor la multi-url pero ahora al entrar ala URL me aparece todo mal, como si no tuviera estilos la página y solo se corrige borrando la "s" del https. 

Espero me puedan ayudar
